lib/azure/service_bus/auth/shared_access_signer.rb

```diff
@@ -6,15 +6,15 @@ module Auth
       class SharedAccessSigner < Azure::Core::Auth::Signer
 
         # The number of seconds from the time of signature that the SAS token will expire
-        attr_accessor :expiry_offset, :key_name
+        attr_accessor :expiry_offset, :key_name, :access_key
```

Comment:
>access_key [](start = 50, length = 10)

Don't you already have an access_key attribute. I'd think we don't need to redefine it again here.